taxi/tests/test_models.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.models import Manufacturer, Driver, Car
+
+
+class TestsModels(TestCase):
+    def test_manufacturer_str(self):
+        manufacturer = Manufacturer.objects.create(name="test", country="Test")
+        self.assertEqual(
+            str(manufacturer),
+            f"{manufacturer.name} {manufacturer.country}"
+        )
+
+    def test_driver_str(self):
+        driver = Driver.objects.create(
+            id="1",

```

Comment:
Specifying an ID during `Driver` object creation is unnecessary unless you have a specific reason for doing so. The database will automatically assign an ID.